pkg/logger/logger.go

```diff
@@ -88,7 +90,7 @@ type Logger struct {
 	stdEnabled     bool
 	authEnabled    bool
 	reqEnabled     bool
-	reverseProxy   bool
+	getClientFunc  GetClientFunc
```

Comment:
Wondering if this should just be `getClient`, I'm 50/50

Reply:
I'm 50/50 too.  Since it's not too long, I think it's better to be explicit?

Reply:
@steakunderscore how do you feel?

Reply:
@steakunderscore ping

Reply:
Normally I would be against stating the type in the variable name, but here I think it's worth it for the clarity it adds.